src/KvnrValidator.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * @author    Davide Danna
+ * @copyright 2020 GAIA AG, Hamburg
+ * @package   KVNR-Validator
+ *
+ * Created using PhpStorm at 07.09.20 10:36
+ */
+
+namespace GaiaGroup;
+
+/**
+ * Class for KVNR code validation
+ */
+class KvnrValidator
```

Comment:
or technically KVNr?

Reply:
I like it

Reply:
I agree it would look nicer, only one observation though: since the repo name is 'gaiagroup/kvnr-validator', shouldn't the class name use same format? I know it's not mandatory, I was just thinking it as a "good practice"

Reply:
I am suggesting renaming the repo to `gaiagroup/kvnr`